# message for face detection 

float64[] distances # distances to faces
float64[] angles # angles to faces